CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
+## [1.1.13] - 2020-12-04
+* Fix on checking error pointer before assigning to prevent crashes
```

Comment:
Might be good clarifying what crashes. 
Some developers might have updated to 1.1.12 already, so we should clarify what 1.1.13 fixes.

Reply:
Added more details